doc/UsersGuide/BuildingRunningTesting/RunSRW.rst

```diff
@@ -629,75 +617,344 @@ The output files (in ``.png`` format) will be located in the ``postprd`` directo
+VX Taskgroup Organization and VX Field Groups
+`````````````````````````````````````````````````
+The VX (meta)tasks in the ``verify_*.yaml`` taskgroup files are described in detail in :numref:`Table %s
+<VXWorkflowTasksTable>`.  They are organized as follows.
+
+The (meta)tasks in ``verify_pre.yaml`` each
+operate on a single obs type (except for ``metatask_check_post_output_all_mems``, which operates on the
+forecast(s) and checks for the presence of all necessary forecast files), while the ones in ``verify_det.yaml``
+and ``verify_ens.yaml`` operate on one or more verification field groups.  A verification field group
+represents one or more meteorologial fields that are operated on (e.g. verified) together in a single
+call to one of the METplus tools (such as GridStat, PointStat, GenEnsProd, and EnsembleStat), and each
+field group has associated with it an obs type against which those forecast fields are verified.  The
+set of valid VX field groups, the obs types they are associated with, and a brief description of the
+fields they include are given in :numref:`Table %s <VXFieldGroupDescsTable>`. 
+
+.. _VXFieldGroupDescsTable:
 
-Users who have already staged the observation data needed for METplus (i.e., the :term:`CCPA`, :term:`MRMS`, and :term:`NDAS` data) on their system should set the path to this data in ``config.yaml``. 
+.. list-table:: Valid Verification Field Groups and Descriptions
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - VX Field Group
+     - Associated Obs Type
+     - Fields Included in Group
+   * - APCP
+     - CCPA
+     - Accumulated precipitation for the accumulation intervals specified in ``VX_APCP_ACCUMS_HRS``
+   * - ASNOW
+     - NOHRSC
+     - Accumulated snowfall for the accumulation intervals specified in ``VX_APCP_ACCUMS_HRS``
+   * - REFC 
+     - MRMS
+     - Composite reflectivity
+   * - RETOP 
+     - MRMS
+     - Echo top
+   * - SFC
+     - NDAS
+     - Various surface and near-surface fields (e.g. at the surface, 2 m, 10 m, etc)
+   * - UPA
+     - NDAS
+     - Various upper-air fields (e.g. at 800 mb, 500 mb, etc)
+
+The list ``VX_FIELD_GROUPS`` in the ``verification:`` section of ``config.yaml`` specifies the VX field
+groups for which to run verification.  Thus, inclusion of a ``verify_*.yaml`` taskgroup file under the
+``rocoto: tasks: taskgroups:`` section of ``config.yaml`` does not mean that all the (meta)tasks in that
+file will necessarily be included in the workflow.  This is because, in order to avoid unwanted computation,
+only those (meta)tasks in ``verify_det.yaml`` and/or ``verify_ens.yaml`` that operate on field groups
+included in ``VX_FIELD_GROUPS`` will appear in the Rocoto XML, and only those (meta)tasks in ``verify_pre.yaml``
+that operate on obs types associated with one of the field groups in ``VX_FIELD_GROUPS`` will appear in 
+the Rocoto XML.  Thus, for example, setting
```

Comment:
```suggestion
The ``VX_FIELD_GROUPS`` list in the ``verification:`` section of ``config.yaml`` specifies the VX field
groups for which to run verification. In order to avoid unwanted computation, the Rocoto XML will include 
only those (meta)tasks that operate on field groups or obs types associated with field groups in ``VX_FIELD_GROUPS``. 
Thus, inclusion of a ``verify_*.yaml`` taskgroup file under the
``rocoto: tasks: taskgroups:`` section of ``config.yaml`` does not mean that all the (meta)tasks in that
file will be included in the workflow. For example, setting:
```

Reply:
Reworded somewhat for concision ;)